gravitee-apim-rest-api/gravitee-apim-rest-api-portal/gravitee-apim-rest-api-portal-rest/src/main/java/io/gravitee/rest/api/portal/rest/resource/AbstractResource.java

```diff
@@ -182,6 +182,10 @@ protected void checkImageFormat(final String encodedPicture) {
             if (!mediaType.startsWith("image/")) {
                 throw new UploadUnauthorized("Image file format unauthorized " + mediaType);
             }
+
+            if (encodedPicture.contains("image/svg")) {
```

Comment:
can't we use the mediaType ?

```suggestion
            if (mediaType.startsWith("image/svg")) {
```